draft-bormann-asdf-instance-information.md

```diff
@@ -269,20 +415,20 @@ sdfInstance:
     scimObjectId: 8988be82-50dc-4249-bed2-60c9c8797677
```

Comment:
Hmm (marker for food for thought):
This says what specifically it is (an identifier for the Thing) and how it is represented in that specific ecosystem (as scimObjectId).
Do we want to disentangle this at some point?
Right now, a different ecosystem cannot detect that scimObjectId actually is an identifier for the Thing and from which ecosystem that is comint.